src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+  @Override
+  public void doWork() {
+    System.out.println("Truck started its work");
```

Comment:
The message 'Truck started its work' could be more informative. According to the checklist, it should indicate what type of `Machine` is working. Consider including the type in the message, such as 'Truck is now working.'